### PR TITLE
Add Contabo icon

### LINKED ISSUE #11496: Request: Contabo
### Brand Name

Contabo

### Website

https://contabo.com

### Popularity Metric

Ranked 29,963 globally on Similarweb. See https://www.similarweb.com/website/contabo.com/#overview.

### Official Resources for Icon and Color

Logo in header.

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by youneskacel**
contabo (2)

**Issue:** closes #11496

**Popularity metric:** similarweb rank : 29,963 
https://www.similarweb.com/website/contabo.com/#overview

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

Used the primary color of the logo

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -3539,6 +3539,12 @@
             "source": "https://www.hashicorp.com/brand",
             "guidelines": "https://www.hashicorp.com/brand"
         },
+        {
+            "title": "Contabo",
+            "hex": "00AAEB",
+            "source": "https://contabo.com/",
+            "guidelines": "https://contabo.com/"
```
Comment (uncenter) [inline _data/simple-icons.json:3546]:
```suggestion
```
Not seeing any guidelines here?

File: icons/contabo.svg
Comment (uncenter) [inline icons/contabo.svg:1]: Hmm it kinda looks from just the naked eye like you squashed the image? Make sure to preserve the aspect ratio of the logo when shrinking to fit the document.

**Review by uncenter — CHANGES_REQUESTED** [review]

**Comment by youneskacel** [comment]
@uncenter I fixed the issues

File: _data/simple-icons.json [hunk 2]
```diff
@@ -3539,6 +3539,11 @@
             "source": "https://www.hashicorp.com/brand",
             "guidelines": "https://www.hashicorp.com/brand"
         },
+        {
+            "title": "Contabo",
+            "hex": "00AAEB",
+            "source": "https://contabo.com/"
```
Comment (uncenter) [inline _data/simple-icons.json:3545]:
```suggestion
            "source": "https://contabo.com"
```

**Review by uncenter — CHANGES_REQUESTED** [review]

File: icons/contabo.svg
Comment (uncenter) [inline icons/contabo.svg:1]: I'm seeing differences compared to the SVG in the website header:

CleanShot 2024-07-21 at 08 06 49@2x

**Review by uncenter — CHANGES_REQUESTED** [review]

File: icons/contabo.svg
Comment (youneskacel) [reply icons/contabo.svg:1]: well it is exactly the same with the one on the header, you can check it has no shadows

Comment (uncenter) [reply icons/contabo.svg:1]: I'm comparing it. The black is your icon, the colorful is the original overlayed on top. It doesn't match up.

Comment (youneskacel) [reply icons/contabo.svg:1]: well as you could see , its a matter of y value , i did it to center it better

Comment (uncenter) [reply icons/contabo.svg:1]: Ah it is. Please make it perfectly centered.

Comment (youneskacel) [reply icons/contabo.svg:1]: you can check now , i fixed it

**Comment by uncenter** [comment]
> Email sent to request permission from Contabo , waiting for their response

As stated in the original issue I've already reached out, so we'll get two responses! 😅

**Comment by youneskacel** [comment]
Permission given by the them

Screenshot 2024-07-29 203040

**Review by LitoMore — APPROVED** [review]

**Pull request merged**